### DISCUSSION TITLE
Make it easier to view pending comments on viewed files

### DISCUSSION THREAD

**Discussion by jchoca** [community/community#12662] [Pull Requests]
I love the ability to mark files as viewed as I'm going through a PR. My workflow is usually:

1) Start review
2) Scroll, viewing one file at a time, marking each as viewed and making comments along the way
3) Submit review

However, sometimes I see something later that makes me want to edit or even delete a pending comment. If that pending comment is on a file that was marked viewed, it may be hard to find. It would be nice if there were at least some indicator that a viewed file had a comment on it. Even better, maybe there could be a sidebar (I see the new file tree view takes up the left side, so maybe it could go on the right side) that displays a summary of the pending comments. Just some ideas :)

**Comment by AnomalRoil** [comment]
Agreed, a visual indicator that viewed files have pending comments would be great.

Also note that currently pending comments do NOT appear in the "Discussion" drop down, which could help with this.

**Comment by pmuellr** [comment]
The scenario gets even more confusing, if you make pending comments in individual commits, and then go back to review of ALL the commits, where you can't see the individual commit comments at all, as near as I can tell.  Then I have to remember which commit I made the comment in.

I would actually like to see a bit than was requested above:

- when "finish your review" popup appears, it should show a list of all the comments I made, maybe just the first few lines if long
- those comments should have a link that would take me to the actual comment, so I can edit/delete from there

**Comment by pmuellr** [comment]
While doing some searching on this scenario, I saw a mention elsewhere with another scenario that I think I've run into.  Multiple reviewing browser sessions open, making comments in both.  I think they kinda don't know about each other, but the "Finish the review" will merge them all (I believe that's what I saw).

Would be nice to either keep these synchronized (which is likely very hard), or just give me one of those "Refresh" buttons like when there are new commits in a PR, so I know things are out-of-sync.

**Comment by ptandler** [comment]
our company recently migrated from gitlab and I miss this feature in github :cry: 

It would be sufficient to include draft comments in the conversations dropdown (in Changed Files tab)

**Comment by lwld** [comment]
Had the same issue, but just found out that you can see the comments if you go back to the "Conversation" tab (= PR main view) and refresh the page, you'll see your pending review comments!

(from here: https://github.com/refined-github/refined-github/issues/3453)
